tests/net-message_test.cpp

```diff
@@ -8,11 +8,10 @@
 namespace tec {
 namespace networking {
 
-// clang-format off
-// Warning: these are picky about whitespace, the s has to go right on the end to make a literal
 #define _STRINGIFY(a) #a
-#define CASE_LABEL(t) case t: return _STRINGIFY(t)s
-// clang-format on
+#define CASE_LABEL(t) \
```

Comment:
Intentional?

Reply:
I removed the "s" from the end intentionally, which is what the format guards were only there for, the old compiler on my Mac wasn't having it.
Then without them, a later run of clang-format insisted on adding the like break: `\`